client/.eslintrc

```diff
@@ -12,8 +12,10 @@
     "sourceType": "module"
   },
   "rules": {
-    "no-console": "error",
-    "react/no-unescaped-entities": "off"
+    "no-console": "off",
```

Comment:
I missed this in the past eslint branch but you'd probably want to enable no-console on client as an extra reminder to remove logs before committing.

Reply:
Can also refer to this for more useful console output to be preserved
https://eslint.org/docs/rules/no-console#options